dev/com.ibm.ws.microprofile.openapi_fat/fat/src/com/ibm/ws/microprofile/openapi/fat/utils/OpenAPITestUtil.java

```diff
@@ -224,6 +224,17 @@ public static void checkPaths(JsonNode root, int expectedCount, String... contai
         expected.stream().forEach(path -> assertNotNull("FAIL: OpenAPI document does not contain the expected path " + path, paths.get(path)));
     }
 
+    public static void checkInfo(JsonNode root, String... requiredInfoFields) {
+        JsonNode infoNode = root.get("info");
+        assertNotNull(infoNode);
+
+        String title = infoNode.get("title").textValue();
```

Comment:
Please add an assertNotNull `infoNode.get("title") and `infoNode.get("version")`.
Otherwise, if any of those fields are null, it would throw NPE.